src/UI/VideoAnalyzer/Events/OSXToKeyScanCodeConverter.cs

```diff
@@ -0,0 +1,171 @@
+//
```

Comment:
Since this is a new file, please the change copyrights header to:
```
//
// Copyright (c) 2010-2022 Antmicro
// Copyright (c) 2022 [Your copyrights]
//
// This file is licensed under the MIT License.
// Full license text is available in 'licenses/MIT.txt'.
//
```

Reply:
Added myself (I left Realtime Embedded since that was in the file upon which this one was based).